tests/scripts/ci/test_fix_markdown_docs.py

```
@@ -0,0 +1,39 @@
+"""Tests for scripts/ci/site/fix-markdown-docs.py."""
+
+from __future__ import annotations
+
+import importlib.util
+from pathlib import Path
+
+_MODULE_PATH = (
+    Path(__file__).resolve().parents[3] / "scripts" / "ci" / "site" / "fix-markdown-docs.py"
+)
+_spec = importlib.util.spec_from_file_location("fix_markdown_docs", _MODULE_PATH)
+assert _spec and _spec.loader
```

Ruff (0.15.14)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)